lib/gap_intelligence/models/merchant_pricing_trend.rb

```diff
@@ -0,0 +1,9 @@
+module GapIntelligence
+  class MerchantPricingTrend < Record
+    attributes :values, :headers
+
+    def initialize(attrs = {})
+      @attributes = attrs
+    end
```

Comment:
actually I do agree, and this means we have a bug in `attr_reader :raw` as it will be changed every time we'll change any instance attributes and raw by meaning will not keep response's raw value